addons/web/static/src/core/file_upload/upload_progress_service.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates>
+    <t t-name="web.UploadProgressManager">
+        <div t-if="state.isVisible or this.props.isVisible" class="o_notification_manager o_upload_progress_toast">
+            <div class="o_notification position-relative show fade mb-2 border border-info bg-white" role="alert" aria-live="assertive" aria-atomic="true">
+                <button type="button" class="btn btn-close o_notification_close p-2" aria-label="Close" t-on-click.prevent="() => cancelAllUpload()"/>
+                <span t-if="state.totalFilesCount" class="text-truncate"><t t-esc="state.totalFilesCount"/> Uploaded</span>
+                <div class="o_notification_body ps-2 pe-4 py-2">
+                    <div class="me-auto overflow-auto o_notification_content" style="max-height: 300px;">
+                        <t t-set="main_file" t-value="this.props.files || state.files"/>
+                        <div t-foreach="main_file" t-as="file" t-key="file" class="o_we_progressbar">
+                            <div class="progressbar">
```

Comment:
Why do we need this, since we already have class="progressbar" in web.ProgressBar?